playbooks/ka-init/group_vars/all.yml

```diff
@@ -18,12 +18,25 @@ crio_version: v1.0.0-rc1
 # - multus
 # - bridge
 # - kokonet-bridge
-pod_network_type: "flannel"
+pod_network_type: "multus"
```

Comment:
Can we leave this as Flannel? Something I would recommend generally is to make a new directory in `./inventory/examples/` with the values you that you require. This will override these defaults, for example see `./inventory/examples/npwg-demo-1/extra-vars.yml`. I'll make other items in this review here with "move to new extra-vars file"

Reply:
+1, too. kube-ansible focuses to install kubernetes with CentOS hence we would provide general CNI for default. Unfortunately multus is not yet...